synthtool/gcp/templates/python_notebooks_testing_pipeline/.cloud-build/cleanup/cleanup-requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-aiplatform
```

Comment:
For most other requirements files, we prefer to pin to specific versions and have them updated by renovate bot. (You can add this non-standard filename to the renovate.json). Is there a reason we're not doing that here?

Reply:
Ooh, I think we just weren't aware of the other standards for synthtool. I'll look into changing this in the PR.